examples/worlds/tunnel.sdf

```diff
@@ -1165,7 +1165,7 @@
                     <outer_angle>1.1</outer_angle>
                     <falloff>1</falloff>
                 </spot>
-                <direction>0 -1 0</direction>
+                <direction>0 0 0</direction>
```

Comment:
I would consider `0 0 0` an invalid direction. i.e, the zero vector has no direction. I'm guessing gz-sim is using the default `0 0 -1`. Can you try that instead?

Reply:
You are correct; it should be 0 0 -1.